concepticondata/conceptlists/Swadesh-1961-100.tsv

```diff
@@ -0,0 +1,101 @@
+ID	NUMBER	SPANISH	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
```

Comment:
instead of ENGLISH, you should write GLOSS, as this is an external translation not attested in teh original source.